pkg/ingester/ingester.go

```diff
@@ -93,6 +93,9 @@ const (
 
 	instanceIngestionRateTickInterval = time.Second
 
+	// Interval for updating resource (CPU/memory) utilization
+	resourceUtilizationUpdateInterval = time.Second
```

Comment:
Given we compute the 1m ewma, do we really need to measure CPU and memory every second? I've the feeling you will get pretty much the same behaviour even measuring it at 5s interval.